2024-02-28T16:10:58.705659077Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-02-28T16:39:42.295779878Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-10T09:16:20.377313322Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("dns error", Custom { kind: Uncategorized, error: "failed to lookup address information: Try again" })) }', src/wunder.rs:23:10
2024-03-15T15:15:41.568902755Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-15T15:22:56.761432261Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: TimedOut }', src/wunder.rs:23:10
2024-03-15T15:28:27.161108727Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-15T15:29:34.880150682Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-03-15T16:09:43.357263951Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-15T16:10:38.612455771Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-15T16:13:21.376085934Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-03-15T16:48:44.801522187Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-03-15T21:31:32.875099747Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("dns error", Custom { kind: Uncategorized, error: "failed to lookup address information: Try again" })) }', src/wunder.rs:23:10
2024-03-15T22:18:22.648026678Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-16T18:21:30.112475280Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-16T18:23:55.244698915Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-17T15:46:59.665315723Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-17T15:47:09.357770855Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-03-19T23:49:22.884170770Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Decode, source: Error("missing field `temperature`", line: 1, column: 236) }', src/web.rs:70:42
2024-03-21T14:07:12.198779887Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:129:50
2024-03-21T14:07:18.836238236Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:129:50
2024-03-21T14:07:27.216905097Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:129:50
2024-03-21T14:08:10.962995824Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:129:50
2024-03-21T14:08:14.415128536Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:129:50
2024-03-23T17:35:45.274924541Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-04-01T15:26:40.392115609Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("dns error", Custom { kind: Uncategorized, error: "failed to lookup address information: Try again" })) }', src/wunder.rs:23:10
2024-04-01T18:30:00.837933585Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-01T18:30:14.529010954Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-01T20:11:22.741724522Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-02T20:11:59.426547627Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-08T00:22:07.480551042Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-08T02:06:54.188851454Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-08T02:09:03.122595444Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-08T02:09:22.372151783Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-04-11T20:41:26.505877558Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-05-29T10:04:09.051476709Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: TimedOut }', src/wunder.rs:23:10
2024-05-29T22:07:53.602426225Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-05-31T19:26:58.951787998Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-03T21:23:02.956510481Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-03T22:12:59.669267238Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-04T14:31:18.801320327Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-06T12:44:37.210790214Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-06T14:39:17.627912274Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-06T20:00:37.925340657Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-06T21:59:19.232064213Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-07T13:10:51.416916192Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-07T19:30:36.929347825Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-07T20:10:14.895887866Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-06-09T00:20:45.453457766Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-07-05T14:20:19.305663839Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T14:45:51.105248807Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T14:52:35.788283902Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T15:10:36.986271084Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T15:14:28.954421217Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T16:40:09.325929818Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T16:48:26.718941493Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T16:50:36.537122852Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" })) }', src/wunder.rs:23:10
2024-07-05T17:11:09.225982936Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Connect, cause: Some(Os { code: 113, kind: HostUnreachable, message: "Host is unreachable" }) }', src/forecast_db.rs:8:62
2024-07-05T17:19:16.359786294Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("dns error", Custom { kind: Uncategorized, error: "failed to lookup address information: Try again" })) }', src/wunder.rs:23:10
2024-07-10T05:56:33.808276404Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, Ssl(Error { code: ErrorCode(1), cause: Some(Ssl(ErrorStack([Error { code: 336151608, library: "SSL routines", function: "ssl3_read_bytes", reason: "tlsv1 alert internal error", file: "ssl/record/rec_layer_s3.c", line: 1562, data: "SSL alert number 80" }]))) }, X509VerifyResult { code: 0, error: "ok" })) }', src/wunder.rs:23:10
2024-07-10T13:05:36.293098824Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-07-10T14:57:26.192788599Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-07-10T15:12:03.885133034Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-07-10T15:19:58.024614540Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-07-11T23:03:49.625231038Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/web.rs:9:55
2024-07-11T23:03:50.447079418Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/web.rs:9:55
2024-07-11T23:04:40.796127110Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/web.rs:9:55
2024-07-11T23:05:06.073532136Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/web.rs:9:55
2024-07-11T23:06:27.124831976Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/web.rs:9:55
2024-07-12T00:48:14.772783080Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/web.rs:9:55
2024-07-12T00:48:16.051294153Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/web.rs:9:55
2024-07-12T16:54:48.356925894Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("dns error", Custom { kind: Uncategorized, error: "failed to lookup address information: Try again" })) }', src/wunder.rs:23:10
2024-07-13T03:48:09.312221125Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E22023), message: "time zone \"gmt-0400\" not recognized", detail: None, hint: None, position: Some(Original(63)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("datetime.c"), line: Some(907), routine: Some("DecodeDateTime") }) }', src/forecast_db.rs:125:10
2024-07-13T03:56:20.767303294Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E22023), message: "time zone \"gmt-0400\" not recognized", detail: None, hint: None, position: Some(Original(63)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("datetime.c"), line: Some(907), routine: Some("DecodeDateTime") }) }', src/forecast_db.rs:125:10
2024-07-13T03:57:23.882983698Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E22023), message: "time zone \"gmt-0400\" not recognized", detail: None, hint: None, position: Some(Original(63)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("datetime.c"), line: Some(907), routine: Some("DecodeDateTime") }) }', src/forecast_db.rs:125:10
2024-07-13T03:57:42.407902930Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E22023), message: "time zone \"gmt-0400\" not recognized", detail: None, hint: None, position: Some(Original(63)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("datetime.c"), line: Some(907), routine: Some("DecodeDateTime") }) }', src/forecast_db.rs:125:10
2024-07-13T03:57:53.934414780Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E22023), message: "time zone \"gmt-0400\" not recognized", detail: None, hint: None, position: Some(Original(63)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("datetime.c"), line: Some(907), routine: Some("DecodeDateTime") }) }', src/forecast_db.rs:125:10
2024-07-13T03:58:23.779574439Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E22023), message: "time zone \"gmt-0400\" not recognized", detail: None, hint: None, position: Some(Original(63)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("datetime.c"), line: Some(907), routine: Some("DecodeDateTime") }) }', src/forecast_db.rs:125:10
2024-07-13T03:58:26.598771732Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E22023), message: "time zone \"gmt-0400\" not recognized", detail: None, hint: None, position: Some(Original(63)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("datetime.c"), line: Some(907), routine: Some("DecodeDateTime") }) }', src/forecast_db.rs:125:10
2024-08-03T21:31:22.496329386Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("api.weather.com")), port: None, path: "/v2/pws/observations/current", query: Some("stationId=KNCSANFO139&format=json&units=e&apiKey=535caee99bf24d7b9caee99bf26d7b9f"), fragment: None }, source: hyper::Error(Connect, ConnectError("dns error", Custom { kind: Uncategorized, error: "failed to lookup address information: Try again" })) }', src/wunder.rs:23:10
2024-08-05T13:28:35.457714668Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-08-05T14:07:09.357112193Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-08-05T14:25:03.091926612Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
2024-08-05T14:25:05.915223577Z  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: PoisonError { .. }', src/main.rs:139:48
